src/ascend.js

```diff
@@ -0,0 +1,29 @@
+var _curry1 = require('./internal/_curry1');
+
+
+/**
+ * Makes an "ascending" comparator function out of a function that returns a value
+ * that can be compared
```

Comment:
- We normally end these blurbs with periods.
- Does ascending need to be quoted?
- Maybe end with "that can be compared with `<` and `>`."